artichoke-backend/Cargo.toml

```diff
@@ -39,6 +39,7 @@ spinoso-securerandom = { version = "0.2.0", path = "../spinoso-securerandom", op
 spinoso-string = { version = "0.20.0", path = "../spinoso-string", features = ["always-nul-terminated-c-string-compat"] }
 spinoso-symbol = { version = "0.3.0", path = "../spinoso-symbol" }
 spinoso-time = { version = "0.7.0", path = "../spinoso-time", features = ["tzrs"], default-features = false, optional = true }
+strftime-ruby = { version = "1.0.0", optional = true }
```

Comment:
`spinoso-time` actually re-exports `strftime-ruby` so I don't think we need to add this direct dependency.

https://artichoke.github.io/artichoke/spinoso_time/#reexports

Reply:
Ah yes, missed this, easy fix!